File: tests/IndirectTest/java/AccessInvokeExceededHelper/src/MyTest.java
import org.junit.TestCase;

public class MyTest extends TestCase {
	A a = new A();
	B b = new B();
	C c = new C();
	D d = new D();
	E e = new E();
	F f = new F();

	public void testCommand() {
		someHelper();
	}

	public void someHelper() {
		a.go();
		b.go();
		c.go();
		d.go();
		e.go();
		f.go();

		int i = 0;
		i = a.attr;
		i = b.attr;
		i = c.attr;
		i = d.attr;
		i = e.attr;
		i = f.attr;
	}
}

File: tests/AssertionRoulette/cpp/SixAssertNoMessage/src/MyTest.cxx
#include "MyTest.h"

void TestStuff::testCommand() {
    CPPUNIT_ASSERT( 1 > 2 );
    CPPUNIT_ASSERT( 2 > 3 );
    CPPUNIT_ASSERT( 3 > 4 );
    CPPUNIT_ASSERT( 4 > 5 );
    CPPUNIT_ASSERT( 5 > 6 );
    CPPUNIT_ASSERT( 6 > 7 );
}
